bodhi/server/config.py

```diff
@@ -475,7 +475,7 @@ class BodhiConfig(dict):
             'value': True,
             'validator': _validate_bool},
         'mail.templates_basepath': {
-            'value': 'bodhi:server/email/templates/',
```

Comment:
I either didn't know or forgot that this was a setting (strange). I noticed that it's not documented in ```production.ini```, so I documented that here: https://github.com/fedora-infra/bodhi/pull/2931

Let's wait to merge this PR until that one is merged, so we can rebase this PR on that one and change that doc to show this new path.

I also want to merge the other one first so I can release it in a 3.y release series (this patch will go to 4.0.0).